types/bootstrap/js/dist/alert.d.ts

```diff
@@ -3,17 +3,31 @@ import BaseComponent from "./base-component";
 declare class Alert extends BaseComponent {
     static NAME: "alert";
     static jQueryInterface: Alert.jQueryInterface;
+
     /**
+     * Closes an alert by removing it from the DOM. If the .fade and .show
+     * classes are present on the element, the alert will fade out before it
+     * is removed.
+     */
+     close(): void;
+
+     /**
+      * Destroys an element's alert. (Removes stored data on the DOM element)
+      */
+     dispose(): void;
```

Comment:
The `BaseComponent` includes `dispose` function. Please remove it from all components.

Reply:
Sorry, I just removed it from all components. Thanks.

Reply:
and already changed and committed, pushed it.